CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+## General guidelines and code of conduct
+
+This repository follows the same [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/digital-asset/daml/blob/master/CODE_OF_CONDUCT.md) as the [main DAML repository](https://github.com/digital-asset/daml/).
+
+## Build and test
+
+This project uses [NPM](https://npmjs.com/) to manage the dependencies and the environment of scripts handling the build lifecycle.
```

Comment:
Should the expected versions of npm and node be documented here? Alternatively, maybe the project should contain a `.nvmrc` or equivalent?

Reply:
Good point, but for the time being the `engines` field in the `package.json` already points at supporting Node.js 8.x and 10.x, all NPM versions shipped with those version of Node.js should work.